packages/main-ui/src/pages/resources/page.rs

```diff
@@ -1,991 +1,1328 @@
-                    class: "flex flex-row h-[40px] justify-center items-center px-[14px] py-[8px] bg-[#2a60d3] rounded-[4px] gap-[5px]",
-                    onclick: move |_| async move {
-                        onupload.call((title(), metadata_urls()));
-                    },
-                    Edit { width: "24", height: "24" }
-                    div { class: "text-white font-semibold text-[#16px]", "{translate.update}" }
+                    RowOption { width: "24", height: "24" }
                 }
-                button {
-                    class: "flex flex-row w-[60px] h-[40px] justify-center items-center bg-white font-semibold text-[#222222] text-[16px]",
-                    onclick: move |e: Event<MouseData>| {
-                        onclose.call(e);
-                    },
-                    "{translate.cancel}"
+                nav { class: "border-2 bg-white invisible border-none shadow-lg rounded w-60 absolute right-0 top-full transition-all opacity-0 group-focus-within:visible group-focus-within:opacity-100 group-focus-within:translate-y-1 group-focus-within:z-20",
+                    ul { class: "py-1",
+                        li {
+                            class: "p-3 text-sm text-gray-700 hover:bg-gray-100 cursor-pointer",
+                            onclick: move |_event| {},
+                            // "{translate.update_material_li}"
+                            "OK"
+                        }
+                        li {
+                            class: "p-3 text-sm text-gray-700 hover:bg-gray-100 cursor-pointer whitespace-nowrap ",
+                            onclick: move |_event| {},
+                            "None"
+                                                // "{translate.remove_material_li}"
+                        }
+                    }
                 }
             }
         }
     }
 }
 
 #[component]
-pub fn CreateMaterialModal(
-    lang: Language,
-    onupload: EventHandler<CreateMetadataRequest>,
-    onclose: EventHandler<MouseEvent>,
-
-    total_types: Vec<String>,
-    total_fields: Vec<String>,
-    total_purposes: Vec<String>,
-    total_resources: Vec<String>,
-    total_authorities: Vec<String>,
-) -> Element {
-    let translate: CreateMaterialModalTranslate = translate(&lang);
-
-    let mut selected_type = use_signal(|| translate.no_selection.to_string());
-    let mut selected_field = use_signal(|| translate.no_selection.to_string());
-    let mut selected_purpose = use_signal(|| translate.no_selection.to_string());
-    let mut selected_source = use_signal(|| translate.no_selection.to_string());
-    let mut selected_authority = use_signal(|| translate.no_selection.to_string());
-
-    let metadata_urls: Signal<Vec<String>> = use_signal(|| vec![]);
-    let mut name: Signal<String> = use_signal(|| "".to_string());
-
+pub fn ResourcePage(props: ResourceProps) -> Element {
+    let translate: ResourceTranslate = translate(&props.lang);
```

Comment:
```suggestion
    let tr: ResourceTranslate = translate(&props.lang);
```
보통 오픈소스에서 개발자들간의 암묵적으로 사용하는 줄임말들이 몇개 있어요. 
예를 들면
String -> str
Conversion -> conv

등이 있는데요. 그 중에 translate 는 tr 로 많이 줄입니다.

알아두면 나중에 코드 볼때 좋으니, 참고하시라고 말씀드려요.

보통 c 계열에서 variable 이름은 3자이하로 적는걸 컨벤션으로 가져가다 보니, 이런것들이 생겼었습니다.